app/models/service_provider.rb

```diff
@@ -17,6 +17,7 @@ class ServiceProvider < ApplicationRecord
 
   enum block_encryption: { 'none' => 0, 'aes256-cbc' => 1 }, _suffix: 'encryption'
   enum identity_protocol: { openid_connect: 0, saml: 1 }
+  enum environment: { 'Testing': 0, 'To go live in production': 1 }
```

Comment:
what if we named the enums for the environments we have?
- `int`
- `staging`
- `prod`
- `dev`